From 36b445b688624039b66dda36cd2a9b4096a9989b Mon Sep 17 00:00:00 2001
From: Neil Horman <nhorman@redhat.com>
Date: Wed, 22 Oct 2014 18:03:09 -0400
Subject: [redhat] configs: add configs to enable building of Huawei cdc modem

Message-id: <1414000989-9522-6-git-send-email-nhorman@redhat.com>
Patchwork-id: 98076
O-Subject: [RHEL7.1 PATCH v2 5/5] redhat: add configs to enable building of Huawei cdc modem
Bugzilla: 1129796
RH-Acked-by: Don Zickus <dzickus@redhat.com>
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>
RH-Acked-by: Dean Nelson <dnelson@redhat.com>

BZ: 1129796
Brew: http://brewweb.devel.redhat.com/brew/taskinfo?taskID=8150596
Testing: Tested by reporter with success
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/configs/generic/CONFIG_USB_NET_HUAWEI_CDC_NCM b/redhat/configs/generic/CONFIG_USB_NET_HUAWEI_CDC_NCM
new file mode 100644
index 0000000..16acb23
--- /dev/null
+++ b/redhat/configs/generic/CONFIG_USB_NET_HUAWEI_CDC_NCM
@@ -0,0 +1 @@
+CONFIG_USB_NET_HUAWEI_CDC_NCM=m
-- 
1.7.1